backend/models/meta.js

```diff
@@ -142,7 +137,7 @@ class Meta {
 	}
 
 	async getMetadataFields(account, model) {
-		const subModelRefs = await Ref.getRefNodes(account, model);
+		const subModelRefs = await getRefNodes(account, model);
```

Comment:
I think we want to get impose history.current here as well.

I don't think we're interested in getting metadataFields form some submodels that are no longer a submodel